libs/community/langchain_community/vectorstores/redis/base.py

```diff
@@ -249,7 +249,7 @@ def __init__(
         redis_url: str,
         index_name: str,
         embedding: Embeddings,
-        index_schema: Optional[Union[Dict[str, str], str, os.PathLike]] = None,
+        index_schema: Optional[Union[Dict[str, list[Dict[str, str]]], str, os.PathLike]] = None,
```

Comment:
IMHO, if you define a type alias, the code will be more understandable and laconic.
Like
`ListOfDictsType = list[Dict[str, str]]`

Reply:
Sure!